src/prepack-cli.js

```diff
@@ -170,6 +174,7 @@ function run(
       }
       process.exit(1);
     }
+    if (serialized == null) serialized = {};
```

Comment:
I've fixed here by adding one more condition to check if `!!serialized`
And log some msg if `serialized.code === ''` which means no input.